torch/csrc/autograd/python_variable.cpp

```diff
@@ -144,6 +144,32 @@ static PyObject *THPVariable_pynew(PyTypeObject *type, PyObject *args, PyObject
 }
 
 // Instantiates a subclass of torch.Tensor. Used by nn.Parameter()
+static PyObject* THPVariable_as_subclass(THPVariable* self, PyObject* args, PyObject* kwargs) {
```

Comment:
Uhh, is this a straight copy paste of `make_subclass`? Can we avoid doing that?

Reply:
It isn't, it's missing the detach and the `requires_grad` calls, and the signature is different. That's for a reason: We want this to be a straight view, not a detached-and-re-attached view as `Parameter` requires.

Reply:
So, you don't want the comment above right?

```
// Instantiates a subclass of torch.Tensor. Used by nn.Parameter()
```